tool/plugin/README.md

```diff
@@ -42,7 +50,10 @@ Only build the specified IntelliJ version. `ID` is one of the "version" strings
 Normally the archive files are not unpacked if the corresponding directory exists. This flag forces the archive files to be unpacked.
 
 
-The make function must generate the correct `plugin.xml` for each platform and generate the correct artifacts for each. In release mode the archive files are always unpacked (--unpack is implied). The plugin files are saved in the `artifacts` directory. In release mode a subdirectory of `artifacts` is created which has the name of the release and they are put there. During a local dev/test run the subdirectory is not created and the created files are children of `artifacts`.
+The make function must generate the correct `plugin.xml` for each platform and generate the correct artifacts for each. In release mode 
+the archive files are always unpacked (--unpack is implied). The plugin files are saved in the `artifacts` directory.
```

Comment:
👍

Reply:
Actually, it's not the `artifacts` dir that's versioned, rather, the extracted contents _in_ the directory. Previously, each subsequent build would have to delete and re-write the `android-studio`, `Dart`, and `idea` directories. Instead, we now unpack to version labeled sub-directories and only re-create if forced (-`u`, `rebuildCache`) or if the directory is older than the zip/tar package.  So I _think_ this can stay 'as is'.  I think there's some ambiguity around 'artifact' as a folder, 'artifact' as the downloaded source/dependencies, and the 'artifact' generated from the build process itself...we can unravel all of that later though.

Reply:
Got it. I got side-tracked again, this time by the new canary release of Android Studio.